demo/App.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react';
+import { Button, Box, Typography } from '@mui/material';
+import Dropzone from 'react-dropzone';
+import { init } from "../lib/main.tsx";
+
+function App() {
+    const [cdeFile, setCdeFile] = useState<File | null>(null);
+    const [datasetFile, setDatasetFile] = useState<File | null>(null);
+
+    const onCdeFileDrop = (acceptedFiles: File[]) => {
+        setCdeFile(acceptedFiles[0]);
+    };
+
+    const onDatasetFileDrop = (acceptedFiles: File[]) => {
+        setDatasetFile(acceptedFiles[0]);
+    };
+
+    const handleRemoveCdeFile = () => {
+        setCdeFile(null);
+    };
+
+    const handleRemoveDatasetFile = () => {
+        setDatasetFile(null);
+    };
+
+    const handleSubmit = (event: React.FormEvent) => {
+        event.preventDefault();
+
+        if (cdeFile && datasetFile) {
+            init({
```

Comment:
Here we have an example of usage of our library. 

It shows which parameters the customer has to provide